model/decoder/panoptic_deeplab.py

```diff
@@ -257,7 +267,11 @@ def call(self, features, training=False):
       The dictionary containing the predictions under the specified key.
     """
     x = self.conv_block(features, training=training)
-    return {self._pred_key: self.final_conv(x)}
+    x = self.final_conv(x)
```

Comment:
Discussion for design choice:

I feel we should still return logits here, instead of tf.sigmoid(x), a particular type of activation. We should add the desired activation at the place where you need it, similar to what we did for tf.nn.softmax(semantic_logits).